src/components/LoadingIndicator/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { ProgressIndicator } from "@nypl/design-system-react-components";
+
+interface LoaderProps {
+  isLoading: boolean;
+}
+
+const LoadingIndicator = ({ isLoading }: LoaderProps) =>
+  isLoading && (
+  <ProgressIndicator
+    isIndeterminate={true}
+    indicatorType="circular"
+    labelText="Loading Indicator"
+    showLabel={false}
+  />
+  );
```

Comment:
We want this loader to be a full page component, it's in a weird position that's breaking the layout. 


<img width="300" height="400" alt="Screenshot 2025-11-10 at 15 02 12" src="https://github.com/user-attachments/assets/df78e8a9-b21b-4f09-b710-ec643216f43f" />